design-at-cornell/src/events/Events.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { PageContainer, EventList } from './EventsStyles';
+import Title from './Title';
+import Filter from './dashboard/Filter';
+import EventDisplay from './dashboard/EventDisplay';
+
+export type Event = {
+  name: string;
+  time: string;
+  location: string;
+  tags: string[];
+  description: string;
+  links: string[];
+};
+
+const Events = () => {
+  const [events] = useState<Event[]>([
+    {
+      name: 'Design Seminar Series with Professor XYZ',
+      time: '3:00 - 4:00 PM EST Saturday, May 1, 2021',
+      location: 'Virtual',
+      tags: ['Annual', 'Virtual', 'Computing in the Arts'],
+      description:
+        'As an expert in spatial design studies, Professor XYZ is visiting to give her annual lecture on how designers can better design for and the description goes on and on but this is not a real event so I’m just typing random words until this text space looks filled enough and this seems good so i’ll stop now.',
+      links: ['google', 'zoom'],
+    },
+    {
+      name: 'Design Seminar Series with Professor XYZ',
+      time: '3:00 - 4:00 PM EST Saturday, May 1, 2021',
+      location: 'Virtual',
+      tags: ['Annual', 'Virtual', 'Computing in the Arts'],
+      description:
+        'As an expert in spatial design studies, Professor XYZ is visiting to give her annual lecture on how designers can better design for and the description goes on and on but this is not a real event so I’m just typing random words until this text space looks filled enough and this seems good so i’ll stop now.',
+      links: ['zoom', 'rsvp', 'facebook'],
+    },
+    {
+      name: 'Design Seminar Series with Professor XYZ',
+      time: '3:00 - 4:00 PM EST Saturday, May 1, 2021',
```

Comment:
Would it be better to separate the time and store it as time, time zone, date (and assuming from date you can get the day)? Or is there a specific reason why you kept it as one string? I'd assume separating it would help with filtering as well, but let me know!